axelrod/load_data_.py

```diff
@@ -1,17 +1,28 @@
+import pathlib
 from typing import Dict, List, Text, Tuple
 
-import os
 import pkg_resources
 
 
-def axl_filename(axl_path: Text) -> Text:
-    """Get the path to Axelrod/<axl_path> from the working directory."""
-    # Working directory
-    dirname = os.path.dirname(__file__)
+def axl_filename(*axl_path: Text) -> Text:
```

Comment:
What about just passing a `pathlib` path object to this function?

So for example the test in `axelrod/tests/integration/test_tournament.py` would become:

```
path = pathlib.Path("test_outputs/test_tournament.csv")
filename = axl_filename(path)
```

The test in the fingerprints file would become:

```
path = pathlib.Path("test_outputs/test_fingerprint.csv")
filename = axl_filename(path)
```

The function code itself would be something like:

```
     axl_path = pathlib.Path(__file__).resolve().parent.parent
     return axl_path/path
```

You could `return str(axl_path/path)` but because of how pathlib handles things I believe you don't need to (I could be wrong).

(I'm also suggesting we change the name of the input variable to be `path`.)

Reply:
sgtm